src/cow-react/modules/swap/services/ethFlow/steps/calculateUniqueOrderId.ts

```diff
@@ -11,9 +13,25 @@ export interface UniqueOrderIdResult {
   orderParams: PostOrderParams // most cases, will be the same as the ones in the parameter, but it might be modified to make the order unique
 }
 
+function incrementFee(params: PostOrderParams): PostOrderParams {
+  const nativeCurrency = params.feeAmount?.currency
+
+  if (!nativeCurrency) {
+    throw new Error('Missing currency for Eth Flow Fee') // Not a realistic case, just to make TS happy
+  }
+
+  const oneWei = CurrencyAmount.fromRawAmount(nativeCurrency, 1)
+  return {
+    ...params,
+    feeAmount: params.feeAmount?.add(oneWei), // Increment fee by one wei
+    sellAmountBeforeFee: params.sellAmountBeforeFee?.subtract(oneWei), // Deduct the sellAmount so the ETH sent is the same
```

Comment:
Still doesn't work https://goerli.etherscan.io/tx/0x34b3edb73eb9a303f37d0a6420ff382142733b4c9d537a260e9a10253d092947

See Nick's comment, it should be `sellAmount` instead of `sellAmountBeforeFee`